fuel-core/src/database.rs

```diff
@@ -443,9 +467,15 @@ mod relayer {
 
         async fn get_sealed_block(
             &self,
-            _height: BlockHeight,
+            height: BlockHeight,
         ) -> Option<Arc<SealedFuelBlock>> {
-            Some(Arc::new(SealedFuelBlock::fix_me_default_block()))
+            let block_id = self
+                .get_block_id(height)
+                .unwrap_or_else(|_| panic!("nonexistent block height {}", height))?;
```

Comment:
Can you simply use `expect` here?
```suggestion
let block_id = self
                .get_block_id(height)
                .expect("nonexistent block height {}", height))?;
```